plugins/source/slack/codegen/tables/templates/tables.go.tpl

```diff
@@ -0,0 +1,18 @@
+// Code generated by codegen; DO NOT EDIT.
+
+package plugin
+
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+{{- range $resource := . }}
+	"github.com/cloudquery/cloudquery/plugins/source/slack/resources/services/{{ $resource.Service }}"
+{{- end }}
+)
+
+func tables() []*schema.Table {
+	return []*schema.Table{
+    {{- range $resource := . }}
```

Comment:
drop extra variable
```
   {{- range . }}
      {{ .Service }}.{{ .TableFuncName }}(),
   {{- end }}
```

Reply:
I think variables serve a purpose for readability - it helps you know what you're working with, and I'd rather not remove it to save a few characters